src/features/api/getMessages.ts

```diff
@@ -27,7 +27,9 @@ export async function handler(
   const result = await client.query<MessagesQueryResult>(query, variables).toPromise();
 
   const multiProvider = await getMultiProvider();
-  const messages = parseMessageQueryResult(multiProvider, result.data);
+  const scrapedChains = await getScrapedChains(client);
```

Comment:
unfamiliar with graphql, will this lead to multiple fetches of domains from the DB / how does the query's `@cached` help?

Reply:
This isn't what the UI uses. This is the handler for the explorer's API, which is actually disabled right now anyway, but I'm trying to keep the code working. It will run once per API request as it should.